dnf-behave-tests/fixtures/specs/kernel/kernel-2.0.0-1.fc29.spec

```diff
@@ -0,0 +1,59 @@
+Name:           kernel
+Epoch:          0
+Version:        2.0.0
+Release:        1.fc29
+
+License:        GPLv2 and Redistributable, no modification permitted
+URL:            https://www.kernel.org/
+
+Summary:        The Linux kernel
+
+Provides:       kernel = %{version}-%{release}
```

Comment:
May I ask you why you have explicit provide of kernel?

Reply:
It was present in our already existing kernel specs, mostly I just copied it.
I have removed it.